Sources/IO/IOStream.swift

```diff
@@ -34,6 +34,10 @@ public protocol IOStreamDelegate: AnyObject {
     func stream(_ stream: IOStream, audioErrorOccurred error: IOAudioUnitError)
     /// Tells the receiver to the stream opened.
     func streamDidOpen(_ stream: IOStream)
+    /// Tells the receiver that the ready state will change.
+    func stream(_ stream: IOStream, willChange readyState: IOStream.ReadyState)
```

Comment:
In Cocoa frameworks, abbreviations like "didChange" or "willChange" are not common, so could you please change them to "didReadyStateChange" or "willReadyStateChange"?

Reply:
I've updated it to "willChangeReadyState" and "didChangeReadyState" as it sounds more readable. Please take a look.